libraries/exoplayer_rtsp/src/main/java/androidx/media3/exoplayer/rtsp/RtpPayloadFormat.java

```diff
@@ -71,6 +80,33 @@ public static String getMimeTypeFromRtpMediaType(String mediaType) {
         return MimeTypes.VIDEO_H265;
       case RTP_MEDIA_MPEG4_GENERIC:
         return MimeTypes.AUDIO_AAC;
+      case RTP_MEDIA_PCM_L8:
+      case RTP_MEDIA_PCM_L16:
+        return MimeTypes.AUDIO_RAW;
+      case RTP_MEDIA_PCMA:
+        return MimeTypes.AUDIO_ALAW;
+      case RTP_MEDIA_PCMU:
+        return MimeTypes.AUDIO_MLAW;
+      default:
+        throw new IllegalArgumentException(mediaType);
+    }
+  }
+
+  /**
+   * Gets the PCM Encoding type for RAW track that is associated with the RTP media type.
+   *
+   * <p>For instance, RTP media type "L8" maps to {@link C.PcmEncoding#ENCODING_PCM_8BIT}.
+   *
+   * @throws IllegalArgumentException When the media type is not supported/recognized.
+   */
+  public static int getPCMEncodingFromRtpMediaType(String mediaType) {
+    switch (mediaType) {
+      case RTP_MEDIA_PCM_L8:
+        // Refer to RFC3551#section-4.5.10
+        return C.ENCODING_PCM_8BIT;
+      case RTP_MEDIA_PCM_L16:
+        // Refer to RFC3551#section-4.5.11
+        return C.ENCODING_PCM_16BIT_BIG_ENDIAN;
       default:
         throw new IllegalArgumentException(mediaType);
```

Comment:
It seems we can reach here if the RTP is MLAW or ALAW? Please word the exception message abit, like 
"Unsupported RTP media type " + mediaType + " in some condition".

Also you can remove the `@throws` in the javadoc, and explain this a bit more in the `<p>` section.

Reply:
In case ALAW/MLAW we set the sampleMimeType as AUDIO_ALAW/AUDIO_MLAW, getPcmEncodingFromAudioEncoding method is called only in case of AUDIO_RAW which we set for L8 or L6. So for MLAW or ALAW cases this method won't be called.